app/controllers/admin/users_controller.rb

```diff
@@ -363,7 +363,7 @@ def set_related_reactions
         Reaction.where(reactable_type: "Comment", reactable_id: user_comment_ids, category: "vomit")
           .or(Reaction.where(reactable_type: "Article", reactable_id: user_article_ids, category: "vomit"))
           .or(Reaction.where(reactable_type: "User", reactable_id: @user.id, category: "vomit"))
-          .includes(:reactable)
```

Comment:
Question: Why do we remove the inclusion of `reactable` here?

Reply:
Removed this as it was leading to `Avoid eager loading` error.
<img width="553" alt="Screenshot 2023-06-09 at 9 43 50 PM" src="https://github.com/forem/forem/assets/9396084/12c03806-e551-43cb-ab4a-fcc3e335a632">